demo/grpc-checks/example/example.go

```diff
@@ -0,0 +1,32 @@
+package example
+
+import (
+	"context"
+	"log"
+
+	// maybe just use the google module?
+	ghc "github.com/hashicorp/nomad/demo/grpc-checks/health"
```

Comment:
If there is an existing module then I agree we use that since I assume thats what most developers would be using.

Reply:
done!